Experience/experience.html

```diff
@@ -0,0 +1,186 @@
+					<li>2014 FIFA World Cup: Dream Team</li>
+					<li>FIFA U-20 World Cup Player of the Tournament: 2005</li>
+					<li>FIFA U-20 World Cup Top Goalscorer: 2005</li>
+					<li>Copa América Young Player of the Tournament: 2007</li>
+					<li>Copa América Top Assist Provider: 2011, 2015</li>
+					<li>Copa América Best Player: 2015</li>
+			</ul>
+		</p>
+
+		
+		<h3 class="experience">	Records </h3>
+		<p class="pexperience">
+			World:
+			<ul>
+					<li>Most FIFA Ballon d'Or awards: 4</li>
+					<li>Guinness World Records title for the most goals in a year: 91 goals</li>
+					<li>Most consecutive European Cup top scorer awards: 4</li>
+					<li>Most Goals Scored in 1 UEFA Champions League Match (joint with Luiz Adriano): 2012</li>
+					<li>Most Hat-Tricks Scored in UEFA Champions League</li>
+					<li>Most Man of the Match Awards in a Single World Cup: 4 (2014)</li>
+			</ul>
+			</p>
+
+		<h3 class="experience">	Spain: </h3>
+		<p class="pexperience">
+			<ul>
+				<li>Most goals scored for a single club: 410 goals
+				<li>Most goals scored in La Liga history: 286 goals
+				<li>Most assists made in La Liga history: 113 assists
+				<li>Most goals scored in La Liga in a season: 50 goals (2011-12)
+				<li>Most consecutive La Liga matches scored in: 21 matches (33 goals)
+				<li>Most La Liga hat-tricks in a season: 8 hat-tricks (shared with Cristiano Ronaldo)
+				<li>Youngest player to score 200 La Liga goals: 25 years
+				<li>First player to score at least 25 goals in five consecutive seasons in La Liga (shared with Cristiano Ronaldo): 2009–10 to 2013–14</li>
+				<li>First player to score at least 20 goals in six consecutive seasons in La Liga: 2008–09 to 2013–14</li>
+				<li>Most goals scored in El Clásico matches: 21 goals</li>
+				<li>Most hat-tricks in El Clásico matches: 2</li>
+				<li>Most goals scored in Derbi Barceloní: 12</li>
+				<li>Most goals scored in the Supercopa de España: 10 goals</li>
+			</ul>
+			</p>
+
+		<h3 class="experience">	Barcelona: </h3>
+		<p class="pexperience">
+			<ul>
+				<li>Most goals scored in El Clásico matches: 21 goals</li>
+				<li>Most goals scored in Derbi Barceloní: 12</li>
+				<li>Top scorer in official competitions: 397 goals</li>
+				<li>Top scorer in La Liga: 286 goals</li>
+				<li>Top scorer in UEFA Champions League: 77 goals</li>
+				<li>Top scorer in international competitions: 80 goals</li>
+				<li>Most hat-tricks in official competitions: 32 hat-tricks</li>
+				<li>Most hat-tricks in La Liga: 24 hat-tricks</li>
+				<li>Most away goals scored in Barcelona history in La Liga: 120 goals</li>
+			</ul>
+		</p>
+
+		<h3 class="experience">	Argentina: </h3>
+		<p class="pexperience">
+			<ul>
+				<li>Most goals scored in a year: 12 goals (shared with Gabriel Batistuta)
+				Youngest Argentinian Player to have Scored in World Cup Finals:
+				18 years and 357 days</li>
+			</ul>
+
+
+		</p>
+		<p><img src="../messi awards.jpg"></p>
+
+
+		<hr class="hr" />
+		Lithgow 2016
```

Comment:
Thanks for all the feedback Aaron. I'll be there on Friday. This past weeks
homework really have me trouble.

-
Luis Lithgow

On Sunday, February 28, 2016, Aaron Cordova <notifications@github.com>
wrote:

> In Experience/experience.html
> <https://github.com/FEWD80/homework-01-famous-person-resume-website/pull/8#discussion_r54360335>
> :
>
> > +
> > +		<h3 class="experience">	Argentina: </h3>
> > +		<p class="pexperience">
> > +			<ul>
> > +				<li>Most goals scored in a year: 12 goals (shared with Gabriel Batistuta)
> > +				Youngest Argentinian Player to have Scored in World Cup Finals:
> > +				18 years and 357 days</li>
> > +			</ul>
> > +
> > +
> > +		</p>
> > +		<p><img src="../messi awards.jpg"></p>
> > +
> > +
> > +		<hr class="hr" />
> > +		Lithgow 2016
>
> [image: :hammer:] Lone text. Wrap this in a div. Best practice is to
> always have text on a page in some element, whether its a <p>, <span>,
> <div> or something else.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/FEWD80/homework-01-famous-person-resume-website/pull/8/files#r54360335>
> .
>


-- 
Thank you,
Luis A. Lithgow
347-213-6088